pom.xml

```diff
@@ -11,6 +11,90 @@
     <properties>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
+        <aspectj.version>1.9.9.1</aspectj.version>
+        <allure.version>2.21.0</allure.version>
+        <allure-maven.version>2.12.0</allure-maven.version>
+        <selenium.version>4.3.0</selenium.version>
+        <junit.version>5.9.0</junit.version>
+        <junit-platform.version>1.7.0</junit-platform.version>
+        <log4j.version>2.19.0</log4j.version>
+        <webdriver-manager.version>5.3.2</webdriver-manager.version>
+        <assertj.version>3.24.2</assertj.version>
     </properties>
-    
+
+    <dependencies>
+
+        <dependency>
+        <groupId>org.seleniumhq.selenium</groupId>
+        <artifactId>selenium-java</artifactId>
+        <version>${selenium.version}</version>
+    </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-launcher</artifactId>
```

Comment:
I think you don't need this dependencies. Try to pull only required dependencies (it will help you with the future maintaining).